client/app/router.js

```diff
@@ -12,4 +12,9 @@ Router.map(function() { // eslint-disable-line
   this.route('projects');
   this.route('login');
   this.route('logout');
+
+  this.route('packages', function() {
```

Comment:
Actually, should this be `package` singular, since this route is editing one specific package?

Reply:
I think this should be plural. A singular package is actually handed in the subroute, packages.edit. Keeping this as "packages" follows the convention in case there is an index subroute here, and multiple packages are needed.

Reply:
sounds good 👍